rule302.md

```diff
@@ -0,0 +1,23 @@
+---
+RULE: 302
+Author: Michael Burns <michael@mirwin.net>
+Status: Draft
+Type: Mutable
+Amends: [203](/rule203.md)
+---
+
+# Rule
+
+Quorum is half of all active, registered [Players](https://github.com/mburns/nomic/wiki/Player). Active is defined as having voting or joined within the last year.
```

Comment:
What if `Active` is defined has having the Player's fork having been updated in the past year. This would mean that they would have had to sync their fork of the main repo to update it recently, but not necessarily have voted.

I think this especially makes sense if quorum is being dropped from 100% to 2/3.